src/main/java/JigaChat.java

```diff
@@ -0,0 +1,85 @@
+import java.util.Scanner;
+public class JigaChat {
+    Task[] taskList = new Task[100];
+    int taskCounter = 0;
+
+    public void readCommand() {
+        String input;
+        String[] commands = new String[2];
+        Scanner in = new Scanner(System.in);
+        input = in.nextLine();
+        commands = input.split(" ", 2);
+
+        switch(commands[0]) {
+        case "bye":
+            System.out.println("Bye. Hope to see you again soon!");
+            System.exit(0);
+        case "list":
+            printList();
+            return;
+        case "mark":
+            System.out.println("JigaChat has marked task " + (Integer.parseInt(commands[1])) + " as done!");
+            taskList[Integer.parseInt(commands[1]) - 1].markAsDone();
+            return;
+        case "unmark":
+            System.out.println("JigaChat has marked task " + (Integer.parseInt(commands[1])) + " as not done!");
+            taskList[Integer.parseInt(commands[1]) - 1].markAsUndone();
+            return;
+        }
+        addToList(commands);
+    }
+
+    public void addToList(String[] taskToAdd) {
```

Comment:
Good use of camelCase and appropriate names for your methods and variables. Explains what you intend to do in the method well. No issues there!